src/main/kotlin/lotto/Play.kt

```diff
@@ -0,0 +1,49 @@
+package lotto
+
+import lotto.domain.LottoPrize
+import lotto.domain.LottoShop
+import lotto.domain.lotto.Lotto
+import lotto.domain.lotto.WinningLotto
+import lotto.view.InputView
+import lotto.view.ResultView
+
+object Play {
+
+    fun start() {
+
+        val userLottoBuyMoney = userLottoBuy()
+
+        val userLottoList = userLottoInput(userLottoBuyMoney)
+
+        val winningLotto = winningLottoInput()
+
+        val totalPrizeMoney = lottoPrize(userLottoList, winningLotto)
+
+        lottoResult(totalPrizeMoney, userLottoBuyMoney)
+    }
+
+    private fun userLottoBuy(): Int = InputView.inputLottoBuyPrice()
+
+    private fun userLottoInput(userLottoBuyMoney: Int): List<Lotto> {
+
+        val userLottoList = LottoShop.createLottoTicket(userLottoBuyMoney)
+
+        ResultView.lottoPurchasesCount(userLottoList.size)
+
+        ResultView.printUserLotto(userLottoList)
+
+        return userLottoList
+    }
+
+    private fun winningLottoInput(): WinningLotto = WinningLotto(InputView.lastWeekLottoNumber())
+
+    private fun lottoPrize(userLottoList: List<Lotto>, winningLotto: WinningLotto): Int {
+        val lottoResult = LottoPrize(userLottoList, winningLotto)
+
+        ResultView.lottoResult(lottoResult.prizeResult)
+
+        return lottoResult.totalPrizeMoney()
+    }
+
+    private fun lottoResult(totalPrizeMoney: Int, userLottoBuyMoney: Int) = ResultView.statistics(totalPrizeMoney, userLottoBuyMoney)
```

Comment:
```suggestion
    private fun lottoResult(totalPrizeMoney: Int, userLottoBuyMoney: Int) = 
        ResultView.statistics(totalPrizeMoney, userLottoBuyMoney)
```
가로로 너무 긴 경우에는 개행을 적절히 활용하는 것도 좋습니다 😄

Reply:
3199ce5 변경했습니다!